src/design-tokens/_widths.yml

```diff
@@ -0,0 +1,8 @@
+aliases:
+  width_xs: '30em'
+  width_s: '40em'
+  width_m: '48em'
+  width_l: '64em'
+  width_xl: '80em'
+  width_xxl: '100em'
+  width_xxxl: '120em'
```

Comment:
Do you think there's an opportunity to make these jumps more consistent?

Right now the gaps between steps are 10, 8, 16, 16, 20, 20.

I went through our old pattern CSS and markup to see which breakpoints we were actually using. Here's what I found...

Step | Value | CSS | Markup
--- | --- | --- | ---
sm | 30em | 22 | 102
md | 48em | 28 | 312
lg | 64em | 22 | 92
xl | 80em | 5 | 0
xxl | 100em | 2 | 0
xxxl | 120em | 2 | 0

What that tells me is that the < 100em values should definitely influence our scale the most. Right now, the scale feels least consistent in the smaller values. It also seems like

Maybe something like this could work?

Step | Ems | Pixels | Notes
--- | --- | --- | ---
xs | 20 | 320 | Old iPhone size, start of responsive font scaling
s | 30 | 480 | Old iPhone landscape
m | 40 | 640 | Maximum prose width, small tablet widths
l | 60 | 960 | Classic "desktop" design width
xl | 80 | 1280 | Approaching classic laptop resolution
xxl | 100 | 1600 | Common desktop monitor resolution
xxxl | 120 | 1920 | HD width

Then the gaps are 10, 10, 20, 20, 20, 20. Crazy-ness?

Reply:
That makes sense to me 👍